clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -62,6 +62,11 @@ const (
 
 	countFastQuery = "sum(increase(%v_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
 
+	// watchListLatencyMetricName the name of the metrics as defined in the api server
+	watchListLatencyMetricName = "apiserver_watch_list_duration_seconds"
+	// watchListLatencyQuery placeholders must be replaced with (1) quantile (2) query window size
+	watchListLatencyQuery = "histogram_quantile(%.2f, sum(rate(%v_bucket{}[%v])) by (group, version, resource, scope, le))"
+
```

Comment:
Looking into test results:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/perf-tests/2764/pull-perf-tests-clusterloader2/1810319004401668096/artifacts/APIResponsivenessPrometheus_simple_load_2024-07-08T14:50:01Z.json

I see only one entry for watchlist (pod list on namespace scope).

Is that expected? What is issueing this request?

Reply:
good question.
do we have metrics so that i could execute the prom query manually ?
could it be that the run was using a server with enabled watchlist feature ?

Reply:
> could it be that the run was using a server with enabled watchlist feature ?

it looks like this was our case, the most recent run doesn't have any entries of watchlist (the feature was turned off on the server some time ago)

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/perf-tests/2764/pull-perf-tests-clusterloader2/1818234814688399360/artifacts/APIResponsivenessPrometheus_simple_load_2024-07-30T11:06:18Z.json

Reply:
So I was wondering why we've seen only one such entry, but I guess it's because it's disabled in client-go by default (and was enabled only in KCM). So that makes sense.